src/NuGet.Services.Validation.Orchestrator/EntityStatusProcessor.cs

```diff
@@ -268,6 +269,14 @@ await _packageFileService.DoesValidationSetPackageExistAsync(validationSet))
                         validationSet.PackageETag);
                 }
 
+                metaData = await _packageFileService.UpdatePackageBlobMetadataInValidationSetAsync(validationSet);
+
+                _logger.LogInformation(
+                        "Update the blob metadata of validation set {ValidationSetId} package {PackageId} {PackageVersion} with SHA512 hash value",
```

Comment:
We may choose to add more metadata to blobs in the future. Consider making this log message a little more generic:

```suggestion
                        "Updated the blob metadata of validation set {ValidationSetId} package {PackageId} {PackageVersion}",
```

Same comment for line 301